samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -143,6 +143,9 @@ LEDs:
    :ref:`zephyr:thingy53_nrf5340` supports only one RGB LED.
    Each RGB LED channel is used as separate LED.
 
+.. note::
+   :ref:`zephyr:thingy53_nrf5340` and :ref:`zephyr:nrf52840dongle_nrf52840` does not support EMDS.
```

Comment:
Why?

Reply:
Neither Thingy or Dongle have the capabilities to support EMDS because it requires an extra capacitor.